src/Display.h

```diff
@@ -2,24 +2,23 @@
 #define DISPLAY_H
 
 #include <string>
+
 #include <SDL2/SDL.h>
 
 class Display
 {
 public:
 	Display(int width, int height, const std::string& title);
+	virtual ~Display();
+
 	void Update();
 	bool IsClosed();
 	void Clear(float red, float green, float blue, float alpha);
-	virtual ~Display();
-protected:
-private:
-	Display(const Display& other);
-	void operator=(const Display& other) {}
 
-	SDL_Window* m_window;
-	bool m_isClosed;
-	SDL_GLContext m_glContext;
+private:
+	SDL_Window* window_;
+	bool isClosed_;
+	SDL_GLContext glContext_;
 };
 
 #endif 
```

Comment:
You should just set the sublime setting to make sure that there is always a new line at the end of each file. Not a big deal but there are lots of files with no new line.